cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go

```diff
@@ -118,7 +118,7 @@ type ClusterConfiguration struct {
 	// FeatureGates enabled by the user.
 	FeatureGates map[string]bool `json:"featureGates,omitempty"`
 
-	// The cluster name
+	// The cluster name	
 	ClusterName string `json:"clusterName,omitempty"`
```

Comment:
there are fmt error, please fix

Reply:
Thank you for your prompt, I have revised it.